worlds/yachtdice/docs/setup_en.md

```diff
@@ -3,19 +3,13 @@
 ## Required Software
 
 - A browser (you are probably using one right now!).
-- Archipelago from the [Archipelago Releases Page](https://github.com/ArchipelagoMW/Archipelago/releases).
 
 ## Playing the game
 Open the Yacht Dice website. There are two options:
-- Download the latest release from [Yacht Dice Release](https://github.com/spinerak/ArchipelagoYachtDice/releases/latest) and unzip the Website.zip. Then open player.html in your browser. 
+- Download the latest release from [Yacht Dice Release](https://github.com/spinerak/ArchipelagoYachtDice/releases/latest) and unzip the Website.zip. Then open index.html in your browser. 
 - Cruise over to the [Yacht Dice website](https://yacht-dice-ap.netlify.app/). This also works on mobile. If the website is not available, use the first option.
```

Comment:
I would reverse these options so the website (the easier way) is listed first.

Maybe explain why one would prefer the one over the other?

Reply:
Changing the order makes sense!
I'm not sure how to explain why it's easier, I think it follows from the fact it's only one step :)
Thanks for the feedback!